core/version/src/db/model.rs

```diff
@@ -0,0 +1,64 @@
+use chrono::NaiveDateTime;
+use self_update::update::Release;
+use serde::{Deserialize, Serialize};
+use std::convert::TryFrom;
+
+use crate::db::schema::version_release;
+
+#[derive(Clone, Debug, Identifiable, Insertable, Queryable, Serialize, Deserialize)]
+#[primary_key(version)]
+#[table_name = "version_release"]
+pub struct DBRelease {
+    pub version: String,
+    pub name: String,
+    pub seen: bool,
+    pub release_ts: NaiveDateTime,
+    pub insertion_ts: Option<NaiveDateTime>,
+    pub update_ts: Option<NaiveDateTime>,
+}
+
+impl DBRelease {
+    pub(crate) fn current() -> anyhow::Result<Self> {
+        Ok(DBRelease {
+            version: ya_compile_time_utils::semver_str().into(),
+            name: ya_compile_time_utils::build_number_str()
+                .map(|n| format!("{} build #{}", ya_compile_time_utils::git_rev(), n))
+                .unwrap_or(ya_compile_time_utils::git_rev().into()),
+            seen: false,
+            release_ts: NaiveDateTime::parse_from_str(
+                ya_compile_time_utils::build_date(),
```

Comment:
I'm not sure about this. It's definitely not the same as github release date.

Reply:
yes, I was also not sure. I will move it to the name string or introduce a new dedicated column

Reply:
but it was added here to enable us to sort via this `release_ts` column (your comment above). And if we hardcode it for fake-entries it will not be possible, that's why I have added second column to sort (comment above).

Reply:
I'm setting `release_ts` to now to be able to sort it properly (comment above)